INSTALL.rst

```diff
@@ -91,19 +92,33 @@ Installation
 Python
 ------
 The HEPData code is only compatible with Python 3 (not Python 2).  It has been tested with Python 3.6.
+It has also been tested with Python 3.8 on an M1 MacBook where some changes were required (documented below).
 
-First install all requirements in a `virtualenv <https://virtualenv.pypa.io/en/stable/installation.html>`_
-using `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/install.html>`_:
+First install all requirements in a `virtualenv <https://virtualenv.pypa.io/en/stable/installation.html>`_.
+(Use `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/install.html>`_ if you prefer.)
```

Comment:
Could you clarify that the instructions below are for using virtualenv directly, not venvwrapper?